modules/tabs-v2/assets/js/editor/widgets/tabs-v2.js

```diff
@@ -0,0 +1,23 @@
+import View from '../views/view';
```

Comment:
Move to the base folder (remove `widgets`)

Reply:
Handled.